### PR TITLE
Update definition of "accessible name"

### PR DESCRIPTION
**Pull request by annethyme**
Adding new paragraph on comparing acc names, that has so far been note in these rules: 
https://auto-wcag.github.io/auto-wcag/rules/SC2-5-3-label-content-name-mismatch.html
https://github.com/auto-wcag/auto-wcag/pull/220
https://github.com/auto-wcag/auto-wcag/pull/251

Closes issue: none, but related to the above

## Guidance for the PR (pull request) creator

**When creating PR:**
- [x] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).

**After creating PR:**
- [x] Add yourself (and co-authors) as "Assignees" for PR
- [x] Add label to indicate if it's a `Rule`, `Definition` or `Chore` (more to the administrative side)
- [x] Add relevant project (e.g. "Q3 2018 Status") to PR
- [ ] OPTIONAL: If you want anyone in particular to review your pull request, assign them as "Reviewers".
- [ ] Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/SC2-5-3-label-content-name-mismatch.md [hunk 1]
```diff
@@ -31,8 +31,6 @@ This rule applies to any element that has:
 
 The complete [visible text content](#visible-text-content) of the target element either matches or is contained within its [accessible name](#accessible-name).
 
-**Note**: Leading and trailing whitespace and difference in case sensitivity should be ignored.
```
Comment (WilcoFiers) [inline _rules/SC2-5-3-label-content-name-mismatch.md]: @annethyme why did you remove this note? Seems pretty important.

Comment (annethyme) [reply _rules/SC2-5-3-label-content-name-mismatch.md]: Because it's in the definition now, since we were writing the same note in many rules.

File: pages/algorithms/accessible-name.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ key: accessible-name
...
 
-The accessible name is calculated using the [accessible name computation](https://www.w3.org/TR/accname).
+The accessible name is calculated using the [accessible name and description computation](https://www.w3.org/TR/accname).
 
-For native markup languages, such as HTML and SVG, additional information on how to calculate the accessible name can be found in https://www.w3.org/TR/html-aam/#accessible-name-and-description-computation and https://www.w3.org/TR/svg-aam/#mapping_additional.
+For native markup languages, such as HTML and SVG, additional information on how to calculate the accessible name can be found in [HTML Accessibility API Mappings 1.0, Accessible Name and Description Computation](https://www.w3.org/TR/html-aam/#accessible-name-and-description-computation] and [SVG Accessibility API Mappings, Name and Description](https://www.w3.org/TR/svg-aam/#mapping_additional).
+
+If deciding whether an accessible name should be considered the same as e.g. another accessible name or a visible name, leading and trailing whitespace and differences in case sensitivity should be ignored.
```
Comment (kasperisager) [inline pages/algorithms/accessible-name.md]: "visible name" is not defined.

**Comment by annethyme** [comment]
I agree with @kasperisager that it would be nice to have these definitions, but this requires changes to several definitions, which I do not find feasible right now with us working hard to implement the first batch of rules. 

I have put back the note on how to compare accessible names into the rule where it was before and removed it from the definition.
Now the only change in this PR is the format of the links.

**Comment by jkodu** [comment]
@annethyme 
Done. Resolved conflicts & merging too.
